plugins/modmail.py

```diff
@@ -193,17 +193,21 @@ async def on_message(self, msg: Message) -> None:
             reference = None
             if thread_id is not None:
                 reference = MessageReference(message_id=thread_id, channel_id=channel.id, fail_if_not_exists=False)
+
             copy_first = None
+
+            if reference is not None:
+                header_copy = await retry(
+                    lambda: channel.send(embed=header, allowed_mentions=mentions, reference=reference), attempts=10
+                )
+            else:
+                header_copy = await retry(lambda: channel.send(embed=header, allowed_mentions=mentions), attempts=10)
+
+            await add_modmail(msg, header_copy)
+
             for content, _ in chunk_messages(items):
-                if reference is not None and copy_first is None:
-                    copy = await retry(
-                        lambda: channel.send(content, allowed_mentions=mentions, reference=reference), attempts=10
-                    )
-                else:
-                    copy = await retry(lambda: channel.send(content, allowed_mentions=mentions), attempts=10)
+                copy = await retry(lambda: channel.send(content, allowed_mentions=mentions), attempts=10)
                 await add_modmail(msg, copy)
-                if copy_first is None:
-                    copy_first = copy
 
             if thread_id is None and copy_first is not None:
                 await retry(lambda: create_thread(msg.author.id, copy_first.id), attempts=10)
```

Comment:
`create_thread` is never called now (it should be), because `copy_first` is always `None`.

Reply:
should be fixed now!